packages/color-scale-block/src/ColorScaleBlock.spec.ct.tsx

```diff
@@ -0,0 +1,567 @@
+            .invoke('attr', 'style')
+            .as('style_1');
+
+        cy.get(COLOR_PICKER_FLYOUT_SELECTOR)
+            .find(COLOR_PICKER_FLYOUT_BRAND_COLOR_SELECTOR)
+            .eq(COLOR_PICKER_FLYOUT_COLOR_INDEX)
+            .click();
+
+        cy.get(COLOR_PICKER_FLYOUT_SELECTOR)
+            .parentsUntil('[role="dialog"]')
+            .find('[data-test-id="button-text"]')
+            .contains('Confirm')
+            .click();
+
+        cy.get(COLOR_SCALE_BLOCK_SELECTOR)
+            .find(COLOR_SQUARE_BACKGROUND_SELECTOR)
+            .eq(COLOR_PICKER_FLYOUT_COLOR_INDEX)
+            .invoke('attr', 'style')
+            .as('style_2');
+
+        cy.get('@style_1').then((style_1) => {
+            cy.get('@style_2').then((style_2) => {
+                expect(style_2).to.include(style_1);
+            });
+        });
+    });
+
+    it('allows the user to delete a color square', () => {
+        const [ColorScaleBlockWithStubs] = withAppBridgeBlockStubs(ColorScaleBlock, {
+            editorState: true,
+            blockSettings: {
+                customHeight: false,
+                heightSlider: '96px',
+                heightInput: '100px',
+                colorInput: [
+                    {
+                        red: 255,
+                        green: 0,
+                        blue: 0,
+                        id: 1,
+                    },
+                ],
+            },
+        });
+
+        mount(<ColorScaleBlockWithStubs />);
+
+        cy.get(COLOR_SCALE_BLOCK_SELECTOR).find(COLOR_SQUARE_BACKGROUND_SELECTOR).should('have.length', 1);
+
+        cy.get(COLOR_SCALE_BLOCK_SELECTOR)
+            .find(COLOR_SQUARE_WRAPPER_SELECTOR)
+            .eq(0)
+            .realHover()
+            .find(COLOR_SQUARE_DELETE_BUTTON_SELECTOR)
+            .eq(0)
+            .click();
+
+        cy.get(COLOR_SCALE_BLOCK_SELECTOR).find(COLOR_SQUARE_BACKGROUND_SELECTOR).should('have.length', 0);
+    });
+
+    it('resizes all color square to take up equal amounts of space in the block when the user clicks on the Resize Evenly button', () => {
+        const [ColorScaleBlockWithStubs] = withAppBridgeBlockStubs(ColorScaleBlock, {
+            editorState: true,
+            blockSettings: {
+                customHeight: false,
+                heightSlider: '96px',
+                heightInput: '100px',
+                colorInput: [
+                    {
+                        red: 255,
+                        green: 0,
+                        blue: 0,
+                        id: 1,
+                    },
+                    {
+                        red: 255,
+                        green: 255,
+                        blue: 0,
+                        id: 2,
+                    },
+                    {
+                        red: 255,
+                        green: 255,
+                        blue: 100,
+                        id: 3,
+                    },
+                ],
```

Comment:
Why are the colors saved in the settings and not within the palette?
There is a method called `createColor` (also `updateColor` and `deleteColor`) in `useColors`

Reply:
Saw the comment from Martin, good for me to keep it then :)